src/http-spec.ts

```diff
@@ -127,7 +127,7 @@ export interface IHttpCookieParam extends IHttpParam {
  */
 
 export interface IHttpContent {
-  schema?: JSONSchema4 | JSONSchema6 | JSONSchema7;
+  schema?: JSONSchema7;
```

Comment:
btw @marcelltoth, I know you've been interested in it.
I eventually decided to go with JSONSchema7 rather than 4, but this will be the actual draft `http-spec` generates, thus it's no longer a lie.

Reply:
Cool! Any version is good that is true 😄